src/main/resources/assets/purpeille/lang/ko_kr.json

```diff
@@ -0,0 +1,190 @@
+    "block.purpeille.polished_ultraviolet_complex": "윤나는 자외선 복합체",
+    "block.purpeille.lightning_block": "금속 번개 블록",
+    "block.purpeille.lightning_slab": "금속 번개 반 블록",
+    "block.purpeille.lightning_stairs": "금속 번개 계단",
+    "block.purpeille.cut_lightning_block": "깎인 금속 번개 블록",
+    "block.purpeille.cut_lightning_slab": "깎인 금속 번개 반 블록",
+    "block.purpeille.cut_lightning_stairs": "깎인 금속 번개 계단",
+    "item.purpeille.impure_preserved_dust": "퍼페일 가루 혼합물",
+    "item.purpeille.preserved_dust": "퍼페일 가루",
+    "item.purpeille.remnant_chunk": "퍼퍼 잔해 조각",
+    "item.purpeille.remnant_brick": "퍼퍼 잔해 벽돌",
+    "item.purpeille.purpeille_ingot": "퍼페일 주괴",
+    "item.purpeille.smoldered_purpeille_ingot": "그을린 퍼페일 주괴",
+    "item.purpeille.ultraviolet_prism": "자외선 프리즘",
+    "item.purpeille.metallic_lightning": "금속 번개",
+    "item.purpeille.encased_core": "캡슐 코어",
+    "item.purpeille.energetic_core": "에너지 코어",
+    "item.purpeille.overclocked_core": "오버클럭 코어",
+    "item.purpeille.vacuous_core": "공허 코어",
+    "item.purpeille.creative_core": "창조 코어",
+    "item.purpeille.purpeille_helmet": "퍼페일 투구",
+    "item.purpeille.purpeille_chestplate": "퍼페일 흉갑",
+    "item.purpeille.purpeille_leggings": "퍼페일 레깅스",
+    "item.purpeille.purpeille_boots": "퍼페일 샌들",
+    "item.purpeille.purpeille_sword": "퍼페일 검",
+    "item.purpeille.purpeille_shovel": "퍼페일 삽",
+    "item.purpeille.purpeille_pickaxe": "퍼페일 곡괭이",
+    "item.purpeille.purpeille_axe": "퍼페일 도끼",
+    "item.purpeille.purpeille_hoe": "퍼페일 괭이",
+    "item.purpeille.ancients_almanac": "고대 역서",
+    "itemGroup.purpeille.main": "퍼페일",
+    "aspect.purpeille.virtuous": "고결한",
+    "aspect.purpeille.excess": "지나친",
+    "aspect.purpeille.heroic": "용감한",
+    "aspect.purpeille.terran": "토착적인",
+    "aspect.purpeille.shocking": "충격적인",
+    "aspect.purpeille.deathly": "치명적인",
+    "aspect.purpeille.limitless": "한없는",
+    "aspect.purpeille.tranquil": "고요한",
+    "aspect.purpeille.unrivaled": "무적의",
+    "revelation.purpeille.spirit": "영혼",
+    "revelation.purpeille.vigor": "활력",
+    "revelation.purpeille.totality": "완전성",
+    "revelation.purpeille.avarice": "탐욕",
+    "revelation.purpeille.malaise": "무기력",
+    "revelation.purpeille.terror": "두려움",
+    "revelation.purpeille.bound": "도약",
+    "revelation.purpeille.pace": "호흡",
+    "revelation.purpeille.immersion": "몰입",
+    "separator.purpeille.warpath": " ",
+    "rite.purpeille.spirit": "Protection hardens the soul.",
+    "rite.purpeille.vigor": "One must learn to control surplus.",
+    "rite.purpeille.totality": "Valiance carries hidden benefits.",
+    "rite.purpeille.avarice": "Draw from the earth what you deserve.",
+    "rite.purpeille.malaise": "A quick poison denies reflection.",
+    "rite.purpeille.terror": "This world is driven by fear.",
+    "rite.purpeille.bound": "Rise to the gods.",
+    "rite.purpeille.pace": "A purposeful walk brings solace.",
+    "rite.purpeille.immersion": "Become deft yet determined.",

```

Comment:
You can remove all translations that keep English text, as that's what the client defaults to. This goes for every English line, not just the highlighted ones